### ISSUE TITLE
NSFW thumbnails not working

### ISSUE THREAD

**Issue by NaturalHate** [QuantumBadger/RedReader#1162]
Inline NSFW previews seem to work fine but when I use thumbnails, as I prefer compact viewing modes, they just vanish and it just shows the title of the submission without any images.

Had this issue with the current and last major versions respectively, the latter was the first one I started using so I don't know how far back this possibly goes.

**Comment by QuantumBadger** [comment]
Thanks for the report, do you have the following preference enabled? `Settings > Images/Video > Show NSFW thumbnails`.

It would also be good to check your [Reddit account settings](https://old.reddit.com/prefs/) to ensure "Hide images for NSFW/18+ content" isn't selected.

**Comment by NaturalHate** [comment]
Yes, the setting is enabled. And I don't have an account anymore.

**Comment by nascentt** [comment]
> Yes, the setting is enabled. And I don't have an account anymore.

I was under the impression you can only view nfsw if you have an account that is a moderator now.

**Comment by NaturalHate** [comment]
> > Yes, the setting is enabled. And I don't have an account anymore.
> 
> I was under the impression you can only view nfsw if you have an account that is a moderator now.

I believe it's specifically porn, which seem to use some sort of algorithm to detect & block it. Regular nsfw tagged content works fine and sometimes explicit stuff slips through. In this case it is a specific issue with the app though, not Reddit itself. Though the bug might be related to being logged in or out, dunno. I can't test it as I can't even create an account now since they also block VPNs, at least on the redesign but IIRC account creation redirects to that for a while now anyway.

**Comment by Edalbegam** [comment]
Disabling 'Blur mature images and media' in [Reddit Feed Settings](https://www.reddit.com/settings/feed) works for me.

**Comment by NaturalHate** [comment]
> Disabling 'Blur mature images and media' in [Reddit Feed Settings](https://www.reddit.com/settings/feed) works for me.

That requires an account.
